Sources/SwiftSyntaxBuilder/Buildables.swift.gyb

```diff
@@ -125,7 +137,12 @@ extension ${kind}Buildable {
 /// ${line}
 %       end
 %     end
+%     protocols = PROTOCOL_CONFORMANCES_MAP.get(node.syntax_kind)
+%     if protocols:
+public struct ${node.syntax_kind}: ${node.base_kind}Buildable, ${', '.join(protocols)} {
```

Comment:
Again, I think we don’t need this if we express all conformances on the `ExpressibleAs` level.